internal/framework/routes.go

```diff
@@ -0,0 +1,98 @@
+package framework
+
+import (
+	"net/http"
+)
+
+type RouteInfo struct {
+	method  string
+	path    string
+	handler http.HandlerFunc
+}
+
+func NewRouteInfo(method string, path string, handler http.HandlerFunc) *RouteInfo {
```

Comment:
It is the main item to store a handler for each route. Maybe it is better to call it Handler?